src/ISLogger.cpp

```diff
@@ -202,6 +234,29 @@ bool cISLogger::InitSaveCommon(eLogType logType, const string &directory, const
     return ISFileManager::PathIsDir(m_directory);
 }
 
+// (DEPRECATED)
```

Comment:
this should be "[[ deprecated ]]"  (which will generate a warning when this function is used).